cscs-checks/libraries/math/trilinos_compile_run.py

```diff
@@ -12,17 +12,15 @@ def __init__(self, linkage):
                               'dom:gpu', 'dom:mc']
 
         # NOTE: PrgEnv-cray in dynamic does not work because of CrayBug/809265
```

Comment:
This note is a bit confusing. You are excluding `PrgEnv-cray` from the `static` variant, but you include it in the `dynamic`, where it fails. I get your point, but I think this note does not belong here. It should be part of the PR description only. As soon as the Cray bug is fixed, this test (without any modification) should pass, shouldn't it? The note will become then obsolete and we would have to fix the test again..

Reply:
This is an error that I introduced in this PR. The dynamic one was not working in the previous PE versions as well.